modules/security-context-constraints-psa-rectifying.adoc

```diff
@@ -0,0 +1,15 @@
+// Module included in the following assemblies:
+//
+// * authentication/managing-security-context-constraints.adoc
+
+:_content-type: PROCEDURE
+[id="security-context-constraints-psa-rectifying_{context}"]
+= Resolving permission errors that result from pod security admission synchronization
```

Comment:
```suggestion
= Resolving permission errors that result from Pod Security admission
```

Reply:
```suggestion
= Resolving permission errors that result from pod security admission 
```

Docs guidelines want lowercase for this.